.github/workflows/pr-stable-build.yml

```diff
@@ -0,0 +1,33 @@
+# allowing test inside PRs
+
+name: "PR build release"
+on:
+    push:
+        branches:
+          - '**'        # matches every branch
+          - '!master'   # excludes master
```

Comment:
What's the reason for this? Afaik our main branch is called `main`, we never actually merge anything into a branch called `master`, right? Therefore, this shouldn't be necessary..

```suggestion
    push:
        branches:
          - '**'        # matches every branch
```

Reply:
fixed in lasts commits

Reply:
Ok got